docs/sources/tempo/operations/best-practices.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Best practices for traces
+menuTitle: Best practices
+description: Learn about the best practices for traces
+weight: 20
+---
+
+# Best practices for traces
+
+This page provides some general best practices for tracing.
+
+## Span and resource attributes
+
+Traces are built from spans. Spans are constructed primarily of span and resource attributes.
+
+A **span attribute** is a key/value pair that exposes context for the span that it exists within. For example, if the span deals with calling another service via HTTP, it could include the HTTP URL (maybe as the span attribute key `http.url`) and the HTTP status code returned (as the span attribute `http.status_code`). Span attributes can consist of varying, non-null types.
+
+Unlike a span attribute, a **resource attribute** is a key/value pair that is concerned around the context of the manner in which the span was collected.
+For example, this could a set of resource attributes concerning a Kubernetes cluster, in which case you may see resource attributes, for example: `k8s.namespace`, `k8s.container_name`, and `k8s.cluster`.
+These can also include information on the libraries that were used to instrument the spans for a trace, or any other infrastructure information.
```

Comment:
```suggestion
A **span attribute** is a key/value pair that exposes context for the span that it exists within. For example, if the span deals with calling another service via HTTP, it could include the HTTP URL (maybe as the span attribute key `http.url`) and the HTTP status code returned (as the span attribute `http.status_code`). Span attributes can consist of varying, non-null types.

Unlike a span attribute, a **resource attribute** is a key/value pair that describes the context of the manner in which the span was collected.
For example, this could be a set of resource attributes concerning a Kubernetes cluster, in which case you may see resource attributes, for example: `k8s.namespace`, `k8s.container_name`, and `k8s.cluster`.
These can also include information on the libraries that were used to instrument the spans for a trace, or any other infrastructure information.
```